README.md

```diff
@@ -98,6 +102,24 @@ commands under `/usr/local/bin`, include files under `/usr/local/include`, etc.
 installation directory name other than `/usr/local` by giving `./configure` the option
 `--prefix=dirname`.  Run `./configure --help` for further details.
 
+### Windows 10 ###
+
+Windows 10 has a "Windows Subsystem for Linux", which essentially allows you to use Linux features in Windows without the need for a dual-boot system or a virtual machine. To activate this, first go to **Settings** -> **Update and security** -> **For developers** and enable developer mode. (This may take a while.) Afterwards, open command prompt and run 
+
+	lxrun /install
+
+This will install the WSL, and afterwards this system can be accessed e.g. by opening command prompt and typing `bash`. With this Linux kernel, installing fplll is then similar to above, except that most likely the package repository is not up to date, and various additional packages need to be installed first. To make sure you only install the most recent software, run:
```

Comment:
Maybe avoid "this Linux kernel" as the Linux kernel is the one thing that isn't there :)